apps/services/user-profile/src/app/v2/me-user-profile.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, UseGuards } from '@nestjs/common'
+import { UserProfileScope } from '@island.is/auth/scopes'
+import { ApiSecurity, ApiTags } from '@nestjs/swagger'
+import { Audit } from '@island.is/nest/audit'
+import { UserProfile } from '../user-profile/userProfile.model'
+import { Documentation } from '@island.is/nest/swagger'
+import { UserProfileService } from './user-profile.service'
+import { UserProfileDto } from './dto/user-profileDto'
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
```

Comment:
~We haven't added ESLint order but we are going to one day, until then we try to manually sort the orders like:

```
import { Controller, Get, UseGuards } from '@nestjs/common'
import { ApiSecurity, ApiTags } from '@nestjs/swagger'

import {
  CurrentUser,
  IdsUserGuard,
  Scopes,
  ScopesGuard,
} from '@island.is/auth-nest-tools'
import type { User } from '@island.is/auth-nest-tools'
import { UserProfileScope } from '@island.is/auth/scopes'
import { Audit } from '@island.is/nest/audit'
import { Documentation } from '@island.is/nest/swagger'

import { UserProfile } from '../user-profile/userProfile.model'
import { UserProfileDto } from './dto/user-profileDto'
import { UserProfileService } from './user-profile.service'
```